src/ewilgs/migrations/0001_initial.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.8 on 2021-11-14 11:42
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
```

Comment:
Can you please add the migration folders to the gitignore? For this to take effect you also need to delete them and commit the changes.